### ISSUE TITLE
Go build fails in test phase

### ISSUE THREAD

**Issue by akalin** [skycocker/chromebrew#87]
I get this:

```
# Testing packages.
fork/exec /tmp/go-build214772872/cmd/addr2line/_test/addr2line.test: permission denied
FAIL    cmd/addr2line   0.001s
?       cmd/cgo [no test files]
fork/exec /tmp/go-build214772872/cmd/fix/_test/fix.test: permission denied
FAIL    cmd/fix 0.001s
fork/exec /tmp/go-build214772872/cmd/go/_test/go.test: permission denied
FAIL    cmd/go  0.001s
fork/exec /tmp/go-build214772872/cmd/gofmt/_test/gofmt.test: permission denied
FAIL    cmd/gofmt       0.001s
fork/exec /tmp/go-build214772872/cmd/nm/_test/nm.test: permission denied
...
```

**Comment by akalin** [comment]
Ah, this is probably because /tmp (and a bunch of other partitions) are mounted as noexec.

**Comment by akalin** [comment]
Ugh, also one of the tests tries to mkdir from /, which fails because it's mounted as read-only.

**Comment by lagerratrobe** [comment]
Confirmed.  I can get past the /tmp problem by remounting it, but the attempt to write at / is still a problem.
```--- FAIL: TestMkdirAllAtSlash (0.00s)
        path_test.go:217: MkdirAll "/_go_os_test/dir": mkdir /_go_os_test: read-only file system```
Since this is in the go tests, not sure how to address this.

**Comment by yoki123** [comment]
I download binary golang package from office website, just look at https://github.com/golang/go/wiki/ChromeOS#install-go

**Comment by lyxell** [comment]
Resolved in #265
